packages/server/src/api/controllers/table/index.ts

```diff
@@ -77,6 +78,7 @@ export async function save(ctx: UserCtx) {
   ctx.eventEmitter &&
     ctx.eventEmitter.emitTable(`table:save`, appId, savedTable)
   ctx.body = savedTable
+  builderSocket.emitTableUpdate(ctx, savedTable)
```

Comment:
It would be beneficial to update the tests to allow us to mock `builderSocket` and confirm it's called with the correct data. The same applies for datasource, rows, etc.

I'm not sure of the performance impact of setting up multiple sockets, but it'd be preferable to avoid this in unit tests